src/quo2/foundations/colors.cljs

```diff
@@ -182,51 +182,42 @@
 (def danger-50-opa-30 (alpha danger-50 0.3))
 (def danger-50-opa-40 (alpha danger-50 0.4))
 
-;;;; Customization
 
-;; Colors for customizing profiles and communities themes
+;; Colors for customizing users account
 (def customization
-  {:primary   {50 primary-50 ;; User can also use primary color as customisation color
-               60 primary-60}
-   :purple    {50 "#7140FD"
-               60 "#5A33CA"}
-   :indigo    {50 "#496289"
-               60 "#3D5273"}
-   :turquoise {50 "#2A799B"
-               60 "#22617C"}
-   :blue      {50 "#2A4AF5"
+  {:blue      {50 "#2A4AF5"
                60 "#223BC4"}
-   :green     {50 "#5BCC95"
-               60 "#4CAB7D"}
    :yellow    {50 "#F6B03C"
                60 "#C58D30"}
-   :orange    {50 "#FF7D46"
-               60 "#CC6438"}
-   :red       {50 "#F46666"
-               60 "#CD5656"}
-   :pink      {50 "#F66F8F"
-               60 "#C55972"}
-   :brown     {50 "#99604D"
-               60 "#805141"}
+   :turquoise {50 "#2A799B"
+               60 "#22617C"}
+   :copper    {50 "#CB6256"
+               60 "#A24E45"}
    :sky       {50 "#1992D7"
                60 "#1475AC"}
+   :camel     {50 "#C78F67"
+               60 "#9F7252"}
+   :orange    {50 "#FF7D46"
+               60 "#CC6438"}
    :army      {50 "#216266"
                60 "#1A4E52"}
+   :pink      {50 "#F66F8F"
+               60 "#C55972"}
+   :purple    {50 "#7140FD"
+               60 "#5A33CA"}
    :magenta   {50 "#EC266C"
-               60 "#BD1E56"}
-   :copper    {50 "#CB6256"
-               60 "#A24E45"}
-   :camel     {50 "#C78F67"
-               60 "#9F7252"}
-   :yin       {50 "#09101C"
-               60 "#1D232E"}
-   :yang      {50 "#FFFFFF"
-               60 "#EBEBEB"}
-   :beige     {50 "#CAAE93"
-               60 "#AA927C"}})
+               60 "#BD1E56"}})
 
 (def colors-map
-  (merge {:danger  {50 danger-50
+  (merge {:beige   {50 "#CAAE93"
```

Comment:
these colors are not in the customization set but are still used by this customs color function so I moved them to this map instead to avoid breaking current implementation. 
imo this requires some future work as we should not be mixing users custom colors with the in app colors (for a function called `custom-color`  👍